.travis.yml

```diff
@@ -59,7 +59,7 @@ jobs:
         sauce_connect: true
 
     - stage: lint
-      script: npm start lint
+      script: npm start lint.ci
```

Comment:
Feels somewhat unintuitive (+ against what a user would expect) for the default `lint` to fix.

What do you think of `reformat.eslint` and `reformat.prettier`?